doc/source/news.rst

```diff
@@ -7,6 +7,29 @@
 News
 ====
 
+.. _release-9-1-2:
+
+Release 9.1.2 - 2020-01-29
+--------------------------
+
+Improvements
+^^^^^^^^^^^^
+
+* [tools] Added a script for copying only files of specify tables or columns.
+
+  * This script name is copy-related-files.rb.
+  * This script useful if we want to extract specifying tables or columns from a huge database.
```

Comment:
Be verb is missing in main sentence.
```diff
-  * This script useful if we want to extract specifying tables or columns from a huge database.
+  * This script is useful if we want to extract specifying tables or columns from a huge database.
```

Reply:
Thank you for your comment.
I've fixed it.